backend/src/main/resources/routes.conf

```diff
@@ -15,13 +15,21 @@ GET        /locations/top/                                     locationControlle
 # Restaurant
 GET        /restaurants/popular                                restaurantController.popularRestaurants
 GET        /restaurants/{id}                                   restaurantController.get
-GET        /restaurants/{id}/reviews                           restaurantController.reviewSize
-GET        /restaurants/{id}/rating                            restaurantController.avgRating
 GET        /restaurants/filter/                                restaurantController.filter
+
 # Review
 GET       /reviews/{id}                                        reviewController.get
 GET       /reviews/restaurant/{id}/rating                      reviewController.ratingForRestaurant
 GET       /reviews/restaurant/{id}/size                        reviewController.numOfReviewsForRestaurant
 POST      /reviews                                             reviewController.create
+
+# Menu
+GET       /menus/{restaurant}                                  menuController.menusByRestuarant
+
+# MenuItems
+GET      /menu-items/{menu}                                    menuItemController.itemsByMenu
+
+# Cousines
+GET      /cousines                                             cousineController.findAll
```

Comment:
no need for this 3 API-s (filter)

Reply:
Fixed in 0cd4debab433260ab6a8267aa0b72b31db581edb